INTRODUCTION.md

```diff
@@ -2161,13 +2161,13 @@ release of librdkafka.
 | 12      | Heartbeat                     | 4          | 3              |
 | 13      | LeaveGroup                    | 5          | 1              |
 | 14      | SyncGroup                     | 5          | 3              |
-| 15      | DescribeGroups                | 6          | 4              |
+| 15      | DescribeGroups                | 6          | 5              |
 | 16      | ListGroups                    | 5          | 4              |
 | 17      | SaslHandshake                 | 1          | 1              |
 | 18      | ApiVersions                   | 4          | 3              |
 | 19      | CreateTopics                  | 7          | 4              |
-| 20      | DeleteTopics                  | 6          | 1              |
-| 21      | DeleteRecords                 | 2          | 1              |
+| 20      | DeleteTopics                  | 6          | 4              |
```

Comment:
We start getting new error `TOPIC_DELETION_DISABLED` from version 3 of `DeleteTopics`. Are we handling this error correctly? If we need a new handling, I would prefer a separate PR for `DeleteTopics`.